spec/system/consumer/split_checkout_spec.rb

```diff
@@ -292,6 +292,30 @@
           expect(page).to have_content "Select a payment method"
         end
       end
+
+      describe "choosing" do
+        shared_examples "bewteen different payment methods" do |pay_method|
+          let!(:payment_method3) { create(:payment_method, distributors: [distributor], name: "Cash") }
+          let!(:payment_method4) { create(:payment_method, distributors: [distributor], name: "BoGuS") }
```

Comment:
I would name these `cash` and `bogus` instead of `payment_method3` and `payment_method4`.

Reply:
Thanks @mkllnk :pray: 

On the payment method re-namings:
- I've renamed `payment_method3` to `cash`
- `payment_method4` was actually a detour, it got removed on the following commit - sorry if this was a bit misleading.